src/components/AlignControl/index.tsx

```diff
@@ -0,0 +1,46 @@
+import { InfoCircleOutlined } from "@ant-design/icons";
+import { Radio, Tooltip } from "antd";
+import React from "react";
+import { createPortal } from "react-dom";
+
+const infoText = `\
+For the associated publication, single cell images were aligned before feature extraction. The \
+apical basal axis of the cell (which is the z-axis in the lab frame of reference) was preserved \
+and we aligned the cells by rotation in the xy-plane. Cells and nuclei are rotated such that the \
+longest cell axis falls along the x-axis.\
+`;
```

Comment:
I wish this string were data. It kinda has no place in this app's source code.  In fact it would be cool if this type of value were encapsulated in a file called "hacksBecauseWeCantMakeThisAppDataDriven.ts" (not literally, but I think there are still a small amount of things like this sprinkled around).  

Maybe a file like extraData.ts containing misc data that could be per dataset:
```
"hipsc_v2022.1":   { "alignInfoText":"...bla bla bla", otherData2: foo, otherData3: bar }
```
(Megan's PR also has some data that would fit in there)

Reply:
Seems appropriate to defer this as well until both PRs are merged.